src/style.css

```diff
@@ -1,3 +1,72 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+:root {
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 60px;
+}
+
+.logo {
+  margin-left: 50px;
+  height: 40px;
+}
+
+ul {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  list-style: none;
+  position: relative;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  text-transform: uppercase;
+  height: 60px;
+  margin-right: 50px;
+}
+
+.nav__list-item {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  color: black;
+  margin-left: 20px;
+  font-size: 12px;
+  height: 60px;
+}
+
+.nav__list-item:hover {
+  color: var(--main-color);
+}
+
+.is-active {
+  color: var(--main-color);
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  font-size: 12px;
+  height: 60px;
+}
+
+.is-active::after {
+  content: "";
+  width: 37px;
```

Comment:
```suggestion
  width: 100%;
```

Reply:
Anastasiia, i fix all that you suggested... But it does't work(((
![image](https://github.com/mate-academy/layout_moyo-header/assets/95960574/fa46930a-8a4d-49cc-9afa-583ed16e1c38)